.github/workflows/ci.yml

```diff
@@ -9,7 +9,19 @@ concurrency:
   cancel-in-progress: true
 jobs:
   lint:
-    uses: inrupt/typescript-sdk-tools/.github/workflows/reusable-lint.yml@v2.0.3
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20"
+          cache: "npm"
+      # Install dependencies:
+      - run: npm ci --ignore-scripts
+      # Build the packages
+      - run: npm run build
+      # Run the linting command:
+      - run: npm run lint
```

Comment:
This is required to stop this linting error https://github.com/inrupt/solid-client-authn-js/actions/runs/5497045924/jobs/10017431247

Reply:
I've opened up https://github.com/inrupt/typescript-sdk-tools/pull/260 which I think is cleaner than my original solution and eslintignoring.